tests/acceptance/cli_test.py

```diff
@@ -984,6 +984,37 @@ def test_run_without_command(self):
             [u'/bin/true'],
         )
 
+    def test_run_rm(self):
+        volumes = self.client.volumes()['Volumes']
+        if volumes is None:
+            volumesBefore = 0
+        else:
+            volumesBefore = len(volumes)
+
+        self.base_dir = 'tests/fixtures/volume'
+        self.dispatch(['run', '--rm', 'test', '/bin/true'])
+        service = self.project.get_service('test')
+        self.assertEqual(len(service.containers(stopped=True, one_off=OneOffFilter.only)), 0)
+
+        volumes = self.client.volumes()['Volumes']
+        if volumes is None:
+            volumesAfter = 0
+        else:
+            volumesAfter = len(volumes)
+
+        self.assertEqual(volumesAfter - volumesBefore, 0)
+
+    def test_run_rm_keeps_named_volume(self):
```

Comment:
Could we do this as part of the same test case (`test_run_rm`) ? A single service with both an anonymous and named volume.